system-config/vagrant-reload.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2013 Aidan Nagorcka-Smith
+#
+# MIT License
```

Comment:
In his repo there was a LICENSE.txt, i just copied those line to the top of this file.